Adjusts production duration functionality

LDQMF opened the pull request:
Adjusted the functionality to make better use of existing functions.

LDQMF committed:
Adjusted to make better use of existing functions

LDQMF committed:
Merge pull request #2 from LDQMF/time_based_on_product

Adjusted to make better use of existing functions

doublechiang reviewed (commented):
@LDQMF  I think you removed the whole fru print command.
Please check how you get logging_durations_secs.

LDQMF commented:
Acknowledge. Will review the issue.

LDQMF commented:
I messaged this on teams, but to keep standard protocol.

Originally, the first commit I added a section to make use of the impitool fru print, but afterwards I realized you already implemented the use of fru print during the initialization of the uut object under the  **_init_uut_from_ipstr(self, ip)** function. So, to make better use of existing code to avoid bloat I added the necessary variable to store the product name to be used as a key in the PRODUCT_DURATION dictionary located in settings.  Afterwards, I removed the section I made in the previous commit. Please advise if I misunderstood any part the existing code base.
